sycl/doc/extensions/IntelGPU/IntelGPUDeviceInfo.md

```diff
@@ -0,0 +1,239 @@
+    }
+
+
+# Intel GPU Number of Subslices per Slice #
+
+A new device descriptor will be added which will provide the number of subslices per slice on an Intel GPU.  If the device is a subdevice, then the number of subslices per slice in the subdevice is returned.
+
+This support will only be provided when using the Level Zero backend \(BE\).  This is the default for Intel GPUs.
+
+## Device Information Descriptors ##
+
+| Device Descriptors | Return Type | Description |
+| ------------------ | ----------- | ----------- |
+| info\:\:device\:\:ext\_intel\_gpu\_subslices\_per\_slice | uint32\_t| Returns the number of subslices per slice.|
+
+
+## Aspects ##
+
+A new aspect, ext\_intel\_gpu\_subslices\_per\_slice, will be added.
+
+
+## Error Condition ##
+
+An invalid object runtime error will be thrown if the device does not support aspect\:\:ext\_intel\_gpu\_subslices\_per\_slice.
+
+## Example Usage ##
+
+Then the number of subslices per slice can be obtained using the standard get\_info() interface.
+
+    if (dev.has(aspect::ext_intel_gpu_subslices_per_slice)) {
+      auto subslices = dev.get_info<info::device::ext_intel_gpu_subslices_per_slice>();
+    }
+
+
+# Intel GPU Number of Execution Units (EUs) per Subslice #
+
+A new device descriptor will be added which will provide the number of EUs per subslice on an Intel GPU.  If the device is a subdevice, then the number of EUs per subslice in the subdevice is returned.
+
+This support will only be provided when using the Level Zero backend \(BE\).  This is the default for Intel GPUs.
+
+## Device Information Descriptors ##
+
+| Device Descriptors | Return Type | Description |
+| ------------------ | ----------- | ----------- |
+| info\:\:device\:\:ext\_intel\_gpu\_eu\_count\_per\_subslice | uint32\_t| Returns the number of EUs in a subslice.|
+
+
+## Aspects ##
+
+A new aspect, ext\_intel\_gpu\_eu\_count\_per\_subslice, will be added.
+
+
+## Error Condition ##
+
+An invalid object runtime error will be thrown if the device does not support aspect\:\:ext\_intel\_gpu\_eu\_count\_per\_subslice.
+
+## Example Usage ##
+
+Then the number of EUs per subslice can be obtained using the standard get\_info() interface.
+
+    if (dev.has(aspect::ext_intel_gpu_eu_count_per_subslice)) {
+      auto euCount = dev.get_info<info::device::ext_intel_gpu_eu_count_per_subslice>();
+    }
+
+
+# Intel GPU Maximum Memory Bandwidth #
+
+A new device descriptor will be added which will provide the maximum memory bandwidth of an Intel GPU.  If the device is a subdevice, then the maximum bandwidth of the subdevice is returned.
+
+This support will only be provided when using the Level Zero backend \(BE\).  This is the default for Intel GPUs.
+
+## Device Information Descriptors ##
+
+| Device Descriptors | Return Type | Description |
+| ------------------ | ----------- | ----------- |
+| info\:\:device\:\:ext\_intel\_max\_mem\_bandwidth | uint64\_t| Returns the maximum memory bandwidth.|
```

Comment:
What are the units of this query?  Bytes per second, bits per second, something else?  That should be part of the documentation.

Reply:
good catch! thanks.